### PR TITLE
Updated GA plugin playgrounds

### PR DESCRIPTION
**Pull request by jboeijenga**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/seven-lions-attack.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-analytics-module-ga4': patch
+'@backstage/plugin-analytics-module-ga': patch
+---
+
+Updated dev playground to make use of registerApi instead of registering the deprecated plugins.
```
Comment (freben) [inline .changeset/seven-lions-attack.md:6]: You can actually remove this file since this is only touching dev setup stuff that doesn't get published.

**Review by freben — COMMENTED** [review]
Excellent! Just need to remove the changeset, then I'm happy to :shipit:

File: .changeset/seven-lions-attack.md (hunk 1 above)
Comment (jboeijenga) [reply .changeset/seven-lions-attack.md]: I removed the file

**Comment by freben** [comment]
Sorry but you forgot to DCO sign that last commit. Check the comment above for instructions.

**Review by vinzscam — APPROVED** [review]
nice!

**Pull request merged**